config.schema.yaml

```diff
@@ -17,6 +17,25 @@ jupyter:
       description: "Token for jupyter.stfc.ac.uk"
       required: false
 
+cloud_names:
+  description: "Cloud name to use in automated jobs"
+  type: "object"
+  required: true
+  properties:
+    dev_cloud:
+      type: "string"
+      description: "Cloud to use when running Automated Jobs on Dev"
+      required: false
+    prod_cloud:
+      type: "string"
+      description: "Cloud to use when running Automated Jobs on Prod"
+      required: false
+    selected_env:
+      type: "string"
+      description: "Cloud to use when running Automated Jobs on Dev"
+      required: true
```

Comment:
`dev_cloud` and `selected_env` have the same descriptions. Are these duplicate properties?

Maybe update the description for `selected_env` to separate it from the `dev_cloud` property?